Library/Formula/wxmac.rb

```diff
@@ -15,6 +15,8 @@ class Wxmac < Formula
   depends_on "jpeg"
   depends_on "libpng"
   depends_on "libtiff"
+  
+  option "with-enable-stl", "use standard C++ classes for everything"
```

Comment:
enable-stl enables: std_iostreams, std_containers and std_string, which cases STL classes to be used instead of certain wx classes in some places. This in turn cases differences at the source code level when using some wx classes. The differences reagarding containers are explained here: http://docs.wxwidgets.org/trunk/overview_container.html

to sum up: even though using stl containers is recommended it might break things for people who are using legacy classes